### ISSUE TITLE
Sketch Extension error export/sync simple file.

### ISSUE THREAD

**Issue by adrisorribas** [avocode/avocode#645]
Trying to export/sync a simple Sketch file to Avocode via extension.

-----
Sketch Exporter 2.1.5 running in Sketch 3.4.2
```
Error: Sketchtool failed to export pages (C936421D-CD47-49B0-96B2-96611CBBD9A7)
    at /Users/xxxxxx/Library/Containers/com.bohemiancoding.sketch3/Data/…/avocode-exporter/lib/image-exporter.js:1:935
    at /Users/xxxxxx/Library/Containers/com.bohemiancoding.sketch3/Data/…/avocode-exporter/lib/image-exporter.js:1:484
    at Object.oncomplete (fs.js:107:15)
```

**Comment by petervandeweerd** [comment]
I have the same issue

Trying to sync a Sketch file to Avocode with extension

Sketch Exporter 2.5.1 running in Sketch 3.5.1


Error: Command failed: 2016-01-28 10:51:28.966 sketchtool[823:326822] Error: caught exception whilst suppressing update events: *** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer)
2016-01-28 10:51:28.967 sketchtool[823:326822] Error: caught exception whilst decoding (version 76):*** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer)
Failed to open document file:///Users/Peter/Desktop/White%20Label/Baixaki/1.%20Home/Home.sketch
((null) MSDocumentDomain:11)

**Comment by rychlis** [comment]
We have released Sketch Extension 3.0.0-prerelease4 with Sketch 3.5 support. Please download the latest version from https://github.com/avocode/avocode/blob/master/sketch-extension-beta.md#installation--update

We're working on updating the older Sketch Plugin 2 however there has been some further issues with Sketch 3.5 unfortunately. You should be able to use the beta version fine though

Follow #609 for more updates about older Sketch plugin, I am closing this issue as a duplicate, thanks for the report
